src/main/webapp/app/modules/account/settings/settings.tsx

```diff
@@ -90,6 +90,52 @@ export class SettingsPage extends React.Component<IUserSettingsProps, IUserSetti
                 }}
                 value={account.email}
               />
+              {/* Organization Name */}
+              <AvField
+                name="organizationName"
+                label={translate('userManagement.organizationName')}
+                placeholder={translate('userManagement.organizationName.placeholder')}
+                type="text"
+                validate={{
+                  required: {
+                    value: true,
+                    errorMessage: translate('register.messages.validate.organizationName.required')
+                  }
+                }}
+                value={account.organizationName}
+              />
+              {/* Organization URL */}
+              <AvField
+                name="organizationUrl"
+                label={translate('userManagement.organizationUrl')}
+                placeholder={translate('userManagement.organizationUrl.placeholder')}
+                type="text"
+                validate={{
+                  maxLength: {
+                    value: 254,
+                    errorMessage: translate('register.messages.validate.organizationUrl.maxlength')
+                  },
+                  pattern: {
+                    value: '^((http|https)://)?(www.)?[0-9a-zA-Z\\-]+\\..+$',
```

Comment:
I am pretty sure that you updated those 2 regex on backend. I guess you should use them instead.

Reply:
It is not required here. On the backend, there was update to allow empty/null options.